### ISSUE TITLE
Client cannot connect to AP after launching rogue AP on easy-creds 

### ISSUE THREAD

**Issue by joeyj2468** [brav0hax/easy-creds#21]
Hello,

I hope you are well.  I have been having an issue in getting easy-creds to work.  I was able to get the prereqs installed, but still having issues.  I am trying to create a honey pot test by creating a free AP as internet is running as well.  I have eth0 as my internet and wlan0 (AWUSO36NH). So, when a user connects they will connect with internet.  Every time client(s) connects to the free AP, it will take a long time to connect.  But when they do connect, internet will not go through and I will not be able to get the credentials as I am wanting (All testing in my home environment) .  Pop-ups when running the free AP come up like sslstrip, ettercap, and dmesg are up.  Also, I am getting an error where "got channel -1, expected a value > 0.  I typed in channel 9.  I know it works on raspberry pi (Kali linux arm) cause I am using a book "Penetration testing Raspberry Pi."  Let me know what you think.  Also, are we suppose to use dhcp3 or isc-dhcp3?

Thanks,
joeyj2468

**Comment by brav0hax** [comment]
Yeah its airbase-ng...it sucks. Need to move everything to hostapd...i
accept pull requests ;)

**Comment by joeyj2468** [comment]
Ok.  Can you please explain hostapd and pull requests? And how to set them up with easy-creds? Been doing this for a whole week to get the fake AP to work. Thanks again man.

-Joeyj2468

**Comment by joeyj2468** [comment]
Also, it keeps saying etho keeps leaving promiscuous mode in dmesg.  Is that a reason why internet is bot coming through? Thanks.

-Joeyj2468
